### PR TITLE
[plugin.video.filmy] v0.1.0

### PR DESCRIPTION
**Pull request by cedkim**
### Description

Filmy is a free, legal, and safe way for kids in Germany to watch live TV.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.filmy/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Filmy für Kodi
```
Comment (enen92) [inline plugin.video.filmy/README.md:1]: English?

File: plugin.video.filmy/addon.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <import addon="xbmc.python" version="2.25.0"/>
+  <import addon="script.module.requests" version="1.1.0"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="main.py">
+  <provides>video</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+  <summary lang="de">Das einzigartiges Fernsehenapp für Kinder, jetzt auf Kodi.</summary>
+  <description lang="de">Filmy ist das einzigartiges (und kostenloses) Fernseh-App für Kinder. Filmy ist sicher und einfach zu benutzen. Man kann auch mit Filmy Live-Sendungen pausieren.</description>
+  <summary lang="en">The new and different live TV app for kids in Germany, now available for Kodi.</summary>
```
Comment (enen92) [inline plugin.video.filmy/addon.xml:16]: en_GB

File: plugin.video.filmy/addon.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  <description lang="en">Filmy is the unique, free, and legal live TV app for kids. It's safe, and provides several useful features.</description>
```
Comment (enen92) [inline plugin.video.filmy/addon.xml:17]: en_GB

File: plugin.video.filmy/addon.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+provider-name="Filmy">
+<requires>
+  <import addon="xbmc.python" version="2.25.0"/>
+  <import addon="script.module.requests" version="1.1.0"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="main.py">
+  <provides>video</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+  <summary lang="de">Das einzigartiges Fernsehenapp für Kinder, jetzt auf Kodi.</summary>
```
Comment (enen92) [inline plugin.video.filmy/addon.xml:14]: de_DE

File: plugin.video.filmy/addon.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  <description lang="de">Filmy ist das einzigartiges (und kostenloses) Fernseh-App für Kinder. Filmy ist sicher und einfach zu benutzen. Man kann auch mit Filmy Live-Sendungen pausieren.</description>
```
Comment (enen92) [inline plugin.video.filmy/addon.xml:15]: de_DE

File: plugin.video.filmy/main.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import sys
+import xbmcgui
+import xbmcplugin
+import requests
+from urlparse import parse_qsl
+
+__url__ = sys.argv[0]
+__handle__ = int(sys.argv[1])
+
+channels = eval(requests.get(url='https://filmy.ml/app/channels.js').text.split("var channels = ")[1].split(";")[0].replace('title', '"title"').replace('description', '"description"').replace('previewImage', '"previewImage"').replace('liveStream', '"liveStream"'))
```
Comment (enen92) [inline plugin.video.filmy/main.py:10]: You shouldn't run eval blindly out of data provided by a third party, this is extremely unsecure. If this is json, use json.loads()

**Comment by enen92** [comment]
Please add a license and define license in add-on.xml

**Comment by enen92** [comment]
Please add a source location in addon.xml

**Comment by enen92** [comment]
Please define a language element in add-on XML? Is the content only German?

File: plugin.video.filmy/README.md (hunk 1 above)
Comment (cedkim) [reply plugin.video.filmy/README.md:1]: The content of Filmy is only German.

File: plugin.video.filmy/main.py (hunk 6 above)
Comment (cedkim) [reply plugin.video.filmy/main.py]: Fixed.

File: plugin.video.filmy/addon.xml (hunk 2 above)
Comment (cedkim) [reply plugin.video.filmy/addon.xml]: Fixed.

File: plugin.video.filmy/addon.xml (hunk 3 above)
Comment (cedkim) [reply plugin.video.filmy/addon.xml]: Fixed.

File: plugin.video.filmy/addon.xml (hunk 4 above)
Comment (cedkim) [reply plugin.video.filmy/addon.xml]: Fixed.

File: plugin.video.filmy/addon.xml (hunk 5 above)
Comment (cedkim) [reply plugin.video.filmy/addon.xml]: Fixed.

**Comment by cedkim** [comment]
I fixed everything! Anything more?

**Comment by flubshi** [comment]
Is filmy.ml a legal content provider? It contains no impressum and currently provides only one german live stream.

**Comment by cedkim** [comment]
It's legal. I made the service, and even though I live in Germany, I'm still learning German. Sorry.

**Comment by flubshi** [comment]
It wasn't up to me to blame you or your language skills. I like that you put a lot of effort in creating such a nice service and in creating a kodi addon!
I am not a lawyer, but I have concerns because your site only supplies (foreign) TV streams.

But it is up to @enen92 to decide...

**Comment by cedkim** [comment]
The service is targeted at users in Germany. Streams don't work outside of Germany.

**Comment by enen92** [comment]
@cedkim unfortunately our german team members pointed out this plugin in our repo is a no-go since it loads a dynamic list of stream we have no control of. At the moment it only provides kika mediathek and we already have an addon for that: https://kodi.wiki/view/Add-on:KiKa_Mediathek

since I'm not

**Comment by cedkim** [comment]
Oh. Well, that's a shame. But, thanks anyways!

**Pull request closed without merging**